File: BackEnd/src/main/java/com/unosquare/admin_core/back_end/dto/EventMessageDTO.java
package com.unosquare.admin_core.back_end.dto;
import lombok.Data;

import java.time.LocalDate;

@Data
public class EventMessageDTO {

    private int eventMessageId;
    private String author;
    private LocalDate lastModified;
    private String message;
    private int messageTypeId;
    private String messageTypeDescription;
}

File: v2/AdminCore.DTOs/EventMessage/EventMessageDto.cs
using AdminCore.DTOs.Employee;
using System;

namespace AdminCore.DTOs.EventMessage
{
  public class EventMessageDto
  {
    public int EventMessageId { get; set; }

    public int EventId { get; set; }

    public EmployeeDto Employee { get; set; }

    public string Message { get; set; }

    public DateTime LastModified { get; set; }
  }
}